synapse/rest/synapse/client/pick_username.py

```diff
@@ -98,6 +98,9 @@ async def _async_render_GET(self, request: Request) -> None:
             "user_attributes": {
                 "display_name": session.display_name,
                 "emails": session.emails,
+                "remote_user_id": map_username_to_mxid_localpart(
```

Comment:
I think these params are documented somewhere (possibly the sample config?), so the doc will need an update

Reply:
Oh good point.